commons/pac-api-commons/src/main/java/com/tmobile/pacman/api/commons/Constants.java

```diff
@@ -573,4 +573,5 @@ public static ExemptionActions fromText(String text){
     } };
 
     String TENABLE_API_URL = "https://cloud.tenable.com";
+    String ALL_SOURCES_ASSET_GROUP = "all-sources";

```

Comment:
is it name? ID? how do we call this value usually?

Reply:
its actually group name